client/cli/src/config.rs

```diff
@@ -41,8 +41,44 @@ pub(crate) const NODE_NAME_MAX_LENGTH: usize = 64;
 /// default sub directory to store network config
 pub(crate) const DEFAULT_NETWORK_CONFIG_PATH: &'static str = "network";
 
+/// Default configuration values used by Substrate
+///
+/// These values will be used by [`CliConfiguritation`] to set
+/// default values for e.g. the listen port or the RPC port.
+pub trait DefaultConfigurationValues {
```

Comment:
@danforbes any ideas of what we could call this instead? :sweat_smile:

Reply:
How's about `CliConfigurationDefaults`?

Reply:
That's actually better :+1: